hivemind/p2p/p2p_daemon.py

```diff
@@ -0,0 +1,45 @@
+import subprocess
+import typing as tp
+
+
+class P2P(object):
+    """
+    Forks a child process and executes p2pd command with given arguments.
+    Sends SIGKILL to the child in destructor and on exit from contextmanager.
+    """
+
+    LIBP2P_CMD = 'p2pd'
```

Comment:
@deniskamazur sanity check: can we guarantee that the p2pd you compile in #153 will be always accessible as P2Pd?

Reply:
Yes, as long as the `main` package stays in the p2pd directory.
That's what `go help install` says:
`Install compiles and installs the packages named by the import paths...```

Reply:
It actually might be a good idea to download a specific version of libp2p-daemon rather than the latest version of the master branch to make sure refactorings of their codebase don't break our builds.